### PR TITLE
Add alerts for kafka burrow exporter

### PR DESCRIPTION
**Pull request by strangeman**

### CODE REVIEW

File: _data/rules.yml [hunk 1]
```diff
@@ -1105,7 +1105,21 @@ groups:
...
                 severity: critical
                 for: 1m
-
+          - name:  linkedin/Burrow 
+            doc_url: https://github.com/linkedin/Burrow
+            rules:
+              - name: Kafka Offset Decreased
+                description: Kafka topic offset was decreased
+                query: 'delta(kafka_burrow_partition_current_offset[1m])<0'
+                severity: high
```
Comment (samber) [inline _data/rules.yml:1114]: Use warning instead ;)

File: _data/rules.yml [hunk 2]
```diff
@@ -1105,7 +1105,21 @@ groups:
...
+                for: 3m
```
Comment (samber) [inline _data/rules.yml:1115]: Does it make sense to wait 3min for this alert to trigger? 🤔

File: _data/rules.yml [hunk 3]
```diff
@@ -1105,7 +1105,21 @@ groups:
...
+              - name: Kafka Consumer Lag
+                description: Lag for consumer exist more than 30 minutes and it increasing
+                query: |
+                  'kafka_burrow_topic_partition_offset - on(partition, cluster, topic) group_right() kafka_burrow_partition_current_offset  >= (kafka_burrow_topic_partition_offset offset 15m - on(partition, cluster, topic) group_right() kafka_burrow_partition_current_offset offset 15m) 
+                    AND kafka_burrow_topic_partition_offset - on(partition, cluster, topic) group_right() kafka_burrow_partition_current_offset > 0'
+                severity: high
```
Comment (samber) [inline _data/rules.yml:1121]: `warning`

**Review by samber — CHANGES_REQUESTED** [review]

File: _data/rules.yml (hunk 2 above)
Comment (strangeman) [reply _data/rules.yml:1115]: Sounds reasonable

File: _data/rules.yml (hunk 1 above)
Comment (strangeman) [reply _data/rules.yml]: Done

**Review by samber — APPROVED** [review]

**Pull request merged**